tools/cibadmin.c

```diff
@@ -221,20 +232,12 @@ read_input(xmlNode **input, GError **error)
  *
  * \param[in]  xml      XML whose digest to output
  * \param[in]  on_disk  If \c true, output the on-disk digest of \p xml
- * \param[out] error    Where to store error
  */
 static void
-output_digest(const xmlNode *xml, bool on_disk, GError **error)
+output_digest(const xmlNode *xml, bool on_disk)
 {
     char *digest = NULL;
 
-    if (xml == NULL) {
-        exit_code = CRM_EX_USAGE;
-        g_set_error(error, PCMK__EXITC_ERROR, exit_code,
-                    "Please supply XML to process with -X, -x, or -p");
```

Comment:
This goes away because we handle it more generally in the new block (`if (!as_xpath)`)